src/theme.ts

```diff
@@ -30,11 +30,11 @@ export const compassTheme = createTheme({
     "0px 1px 3px 1px rgba(0, 0, 0, 0.15), 0px 1px 2px 0px rgba(0, 0, 0, 0.30)",
     "0px 2px 6px 2px rgba(0, 0, 0, 0.15), 0px 1px 2px 0px rgba(0, 0, 0, 0.30)",
     "0px 1px 3px 0px rgba(0, 0, 0, 0.30), 0px 4px 8px 3px rgba(0, 0, 0, 0.15)",
-    "0px 2px 3px 0px rgba(0, 0, 0, 0.30), 0px 6px 10px 4px rgba(0, 0, 0, 0.15)",
+    "0px 2px 3px 0px rgba(141, 59, 59, 0.3), 0px 6px 10px 4px rgba(0, 0, 0, 0.15)",
```

Comment:
Why are the RGB values for the color changed here?

Reply:
All the other changes look good to me, and they resolve the warnings that `npm run lint` shows in the main branch. ✅

Reply:
Same question, otherwise looks good to me too.

Reply:
good catch, thank you! no idea. maybe a color picker misclick.